File: LargestNum/LargestNum.py
# Find out Largest number amoung 3 numbers.
# a = int(input("Enter the first number: "))
# b = int(input("Enter the second number: "))
# c = int(input("Enter the third number: "))

# if a >= b and a >= c:
#     print("The largest number is:", a)
# elif b >= a and b >= c:
#     print("The largest number is:", b)
# else:
#     print("The largest number is:", c)
 
  
# --------------------------------

# Find out Largest number amoung 3 numbers.
   
a = int(input("Enter 1st number: "))
b = int(input("Enter 2nd number: "))
c = int(input("Enter 3rd number: "))

if a == b == c:
    print(f'All numbers are equal. a = {a}, b = {b}, c = {c}')
elif a == b and a > c:
    print(f'Both numbers are equal and largest: a = {a}, b = {b}')
elif a == c and a > b:
    print(f'Both numbers are equal and largest: a = {a}, c = {c}')
elif b == c and b > a:
    print(f'Both numbers are equal and largest: b = {b}, c = {c}')
elif a > b and a> c:
    print(f'a = {a} is the largest number')
elif b > a and b > c:
    print(f'b = {b} is the largest number')
else: 
    print(f'c = {c} is the largest number')

File: LargestNum/LargestNum.cpp
// Find out Largest number amoung 3 numbers.

// #include <iostream>
// using namespace std;

// int main() {
//     int a, b, c;

//     cout << "Enter three numbers: ";
//     cin >> a >> b >> c;

//     if (a >= b && a >= c) {
//         cout << "The largest number is: " << a << endl;
//     } else if (b >= a && b >= c) {
//         cout << "The largest number is: " << b << endl;
//     } else {
//         cout << "The largest number is: " << c << endl;
//     }

//     return 0;
// }

// --------------------------------------------

// Find out Largest number amoung 3 numbers.

#include <iostream>
using namespace std;
int main()
{
    int a, b, c;

    cout << "Enter the first number: ";
    cin >> a;
    cout << "Enter the second number: ";
    cin >> b;
    cout << "Enter the third number: ";
    cin >> c;
    if (a == b && b == c)
    {
        cout << "All numbers are equal: a= " << a << ", b = " << b << ", c = " << c << endl;
    }
    else if (a == b && a > c)
    {
        cout << "Both numbers are equal and largest: a = " << a << ", b = " << b << endl;
    }
    else if (a == c && a > b)
    {
        cout << "Both numbers are equal and largest: a = " << a << ", c = " << c << endl;
    }
    else if (b == c && b > a)
    {
        cout << "Both numbers are equal and largest: b = " << b << ", c = " << c << endl;
    }
    else if (a > b && a > c)
    {
        cout << "The largest number is: a = " << a << endl;
    }
    else if (b > a && b > c)
    {
        cout << "The largest number is: b = " << b << endl;
    }
    else
    {
        cout << "The largest number is: c = " << c << endl;
    }
    return 0;
}